lib/tasks/coverage_report.rake

```diff
@@ -7,15 +7,20 @@ namespace :coverage do
   desc "Collates all result sets generated by the different test runners"
   task :report do
     require 'simplecov'
+    require 'fileutils'
 
     SimpleCov.coverage_dir('coverage_results')
 
     if ENV['CI']
-      SimpleCov.collate(Dir["*/coverage_*/.resultset.json"]) do
+      SimpleCov.collate(Dir['*/coverage_*/.resultset.json']) do
         refuse_coverage_drop
       end
     else
-      SimpleCov.collate(Dir["lib/coverage_*/.resultset.json"])
+      SimpleCov.collate(Dir['lib/coverage_*/.resultset.json'])
     end
+
+    coverage_dirs = Dir['lib/coverage_*']
```

Comment:
Thank you! I was trying to figure out how to negate it!!

Reply:
Hmm... so I tweaked this a bit to read `Dir['lib/coverage_{[!results]*}'].each { |dir| FileUtils.rm_rf(dir) }`, and it deleted most of the directories, but has left one or two behind. Any idea why?

Here's an example of one of the directory names that remained: `lib/coverage_ec594bfd-f977-4ca9-98a3-38f2899548cb`

Reply:
Whoops! My mistake. By using `[!report]`, I inadvertently told `Dir` to ignore any string containing an `r', `e`, `p`, `o`, `r`, or `t` instead of telling it ignore any string containing `report`.

To do the latter, the correct syntax would be:

```ruby
Dir['lib/coverage_{[!r][!e][!s][!u][!l][!t][!s]}*']
```

Which is looking a bit funky.

Reply:
That works! Thanks for your help! 😄